styles/styles.css

```diff
@@ -48,32 +62,79 @@ body {
     font-weight: 700;
     cursor: pointer;
 }
+.more_about_company:hover {
```

Comment:
It's better to use `-` instead of `_` for class names
`.more-about-company:hover`

Reply:
done